ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -2159,6 +2159,8 @@ periodics:
         value: stable
       - name: CLOUD_PROVIDER_REGION
         value: us-east-2
+      - name: CLUSTER_VERSION
+        value: openshift-v4.8.21
```

Comment:
Does this need to be hard coded? I can imagine this can become quite obsolete given how often the OSD patches are being delivered.

Reply:
That is a good point, I forgot to explain it in this PR. I was trying to find out if there is a way to specify just major and minor version of OSD (the latest patch version would be picked up automatically). It should be possible according to osde2e docs, but doesn't seem to be working ([link to slack conversation](https://coreos.slack.com/archives/CMK13BP4J/p1638451509109100))

Reply:
Cool, thank you. So it seems we need to wait till https://issues.redhat.com/browse/SDCICD-722 is solved.